oz/error_pages/__init__.py

```diff
@@ -41,7 +40,8 @@ def get_lines_from_file(filename, lineno, context_lines):
     Returns (pre_context_lineno, pre_context, context_line, post_context).
     """
 
-    get_lines = lambda s, e: [linecache.getline(filename, l).rstrip() for l in range(s, e)]
```

Comment:
Yeah, we just had this same discussion in a PR for one of our internal projects, and the conclusion came down to "we use lambda pervasively, it hasn't been a problem, so it's fine to go against pep8 on this one like we do on occasion." Though thinking about it, I suppose that logic doesn't necessarily apply to our open source code. We can keep this change.